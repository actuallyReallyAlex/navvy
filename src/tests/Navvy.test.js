import React from 'react'
import Navvy from '../index'
import { shallow } from 'enzyme'

const defaultProps = {
  sections: [
    {
      heading: 'Navigation',
      links: [
        { title: 'Dashboard' },
        {
          title: 'Page layouts',
          handler: () => console.log('LOL')
        },
        { title: 'Navigation' },
        { title: 'Widget' }
      ]
    },
    {
      heading: 'UI Element',
      links: [
        { title: 'Basic' },
        { title: 'Advance' },
        { title: 'Extra' },
        { title: 'Animations' },
        { title: 'Icons' }
      ]
    },
    {
      heading: 'Forms',
      links: [
        { title: 'Form' },
        { title: 'Form Picker' },
        { title: 'Form Select' },
        { title: 'Form Masking' },
        { title: 'Form Wizard' }
      ]
    }
  ]
}

describe('<Navvy />', () => {
  afterEach(() => {
    // defaultProps.handler.mockClear()
  })

  test('Should render the <Navvy /> component.', () => {
    const component = shallow(<Navvy {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
