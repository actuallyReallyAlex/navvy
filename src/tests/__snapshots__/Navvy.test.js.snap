// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navvy /> Should render the <Navvy /> component. 1`] = `ShallowWrapper {}`;
